IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormFileHistoryTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using CommonTestUtils;
+using CommonTestUtils.MEF;
+using FluentAssertions;
+using GitCommands;
+using GitUI;
+using GitUI.CommandsDialogs;
+using GitUIPluginInterfaces;
+using Microsoft.VisualStudio.Composition;
+using NUnit.Framework;
+
+namespace GitExtensions.UITests.CommandsDialogs;
+
+[Apartment(ApartmentState.STA)]
+public class FormFileHistoryTests
+{
+    [SetUp]
+    public void SetUp()
+    {
+        AppSettings.UseBrowseForFileHistory.Value = false;
+
+        var composition = TestComposition.Empty
```

Comment:
Our preferred coding style is:
```suggestion
        TestComposition composition = TestComposition.Empty
```